File: Data Structures & Algorithms/number-of-senior-citizens/submission-1.cpp
class Solution {
public:
    int countSeniors(vector<string>& details) {
        
        int cnt=0;
        int n=details.size();
        for(int i=0;i<n;i++){
            
            int a=(details[i][11]-'0');
            int b=(details[i][12]-'0');

            if((a*10+b)>60) cnt++;
            
            

        }

        return cnt;

    }
};

File: Data Structures & Algorithms/time-needed-to-buy-tickets/submission-1.java
class Solution {
    public int timeRequiredToBuy(int[] tickets, int k) {
         int cnt=0;
         while(tickets[k]!=0){


            for(int i=0;i<tickets.length;i++){
                
                if(tickets[i]!=0){
                    cnt++;
                    tickets[i]--;
                }
                if(tickets[k]==0){
                    return cnt;
                }


            }






         }
         return cnt;


    }
}